deploy/charts/templates/podmonitors.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
```

Comment:
what is this doing?

Reply:
Its useful if user dont install our `kube-prometheus-stack`. They bring their own. However, wants our rules and dashboard to get installed. In that case when the template functions runs it will check if `kube-prometheus-stack` CR is installed or not before forming CR objects.

.Capabilities.APIVersions is a built in object . During install/update helm reaches out to k8s cluster to get all apiVersions installed.

https://helm.sh/docs/chart_template_guide/builtin_objects/